src/docs/howto-users-and-authentication/setup-oidc-proxy/index.md

```diff
@@ -78,43 +62,34 @@ at tools like [Ngrok](https://ngrok.com/) or
 [Localtunnel](https://localtunnel.github.io/www/) that allow exposing
 your local ports publicly.
 
-## Create a login app in LinkedIn
+## Create an OAuth app in Github
 
 1. Head to
-   [LinkedIn apps management page](https://www.linkedin.com/developers/apps/new).
-   A LinkedIn account is required.
-1. Click "Create app"
-1. Add app name
-1. Search the LinkedIn page of your marketplace business. If you do not
-   have a LinkedIn page, you can create one by selecting "+ Create a new
-   LinkedIn Page".
-1. Fill in the URL to the privacy policy in your marketplace.
-1. Add a logo.
-1. Check the legal agreement and you are ready to click "Create app"
-1. Navigate to the _Auth_ tab in your new app view.
-1. Add a new redirect URL:
-   `<your marketplace URL>/api/auth/linkedin/callback`. So for example,
-   `https://www.mymarketplace.com/api/auth/linkedin/callback`
-1. Make a note of the client ID and client secret. You will need these
-   values later on.
-1. Move to the _Products_ tab and add _Sign In with LinkedIn_ by
-   clicking "Select" by the product.
-1. It takes a few moments for LinkedIn to validate your app for the
-   _Sign In_ product.
+   [Github develoepr settings](https://github.com/settings/developers).
+   A Github account is required.
+1. Select "OAuth apps" in the left sidebar, and click "Register a new
+   application".
+1. Add application name and homepage URL.
```

Comment:
Can this be any, arbitrary URL?

Reply:
Good catch! This URL identifies the authenticating app to the end user. I've added a clarification and a screenshot.